data/kubemacpool/kubemacpool.yaml

```diff
@@ -285,6 +285,12 @@ spec:
             configMapKeyRef:
               key: RANGE_END
               name: kubemacpool-mac-range-config
+        - name: CA_ROTATE_INTERVAL
+          value: 8760h0m0s
+        - name: CA_OVERLAP_INTERVAL
+          value: 8760h0m0s
```

Comment:
This is the same as rotate interval, how does that work? Do we always have 2 CAs?

Reply:
This is the previous behaviour where CAs live at CABunlde until expiration time, but we can change if we want, in fact I suppose we have to make this numbers similar to the other kubevirt projects.

Reply:
Ok, if it matches the previous behavior let's be it. It looks odd, but it will be changed pretty soon anyway.